presentations/en/SLF.md

```diff
@@ -0,0 +1,75 @@
+# Senior Leadership Forum
+
+ESDC IT Strategy (30min presentation)
+
+## Agenda
+
+1. (5 min) Approach to IT Strategy
+2. (5 min) Team's focus areas
+3. (5 min) Recommended immediate actions
+
+(15 min) Q&A and buffer time
+
+---
+
+## 1. IT Strategy approach
+
+Purpose: Set the stage by clearly communicate that we are not doing conventional IT strategy with some examples of what that means
+
+Talk about:
+
+- Genesis of IT strategy and reason why the team got created (Dec 2018 senior leader meeting, goal of getting an integrated business-IT strategy)
+- How we are **NOT** doing this
+- We instead will focus on IT's ability to quickly respond to business changes
+- Talk about premise: the relationship between business and IT MUST change to a shared accountability one as the conventional contracting model (requirements based, a company within a company) of business-IT does not work and must move to a partnership one (outcomes based, one team). The future is too unpredictable and complex so the conventional long term planning do not work, we must instead reward an empirical cycle of trying, observing, and correcting.
+
+Quote of Mark Schwartz
+> To work effectively in the digital world, you must first accept complexity and uncertainty, for they demand very different approach to carrying out initiatives. A predictable world rewards advanced planning and rigid plan execution. But a complex and uncertain world rewards an empirical cycle of trying, observing, and correcting.
+
+Talk about 4 angles the team is addressing using strategies (mandate)
+
+- Diagnose and Recommend changes on problems that inhibit the desired changes in behaviours
+  - example: Heavy Governance promotes risks, need to reduce burden of giving permission to work
```

Comment:
Flip the control with guardrails, auditable solutions (Git offers full view of changes and by whom), automated compliance (CI tests), etc.